include/boost/wave/util/cpp_macromap_utils.hpp

```diff
@@ -343,6 +343,27 @@ is_whitespace_only (ContainerT const &argument)
     return true;
 }
 
+///////////////////////////////////////////////////////////////////////////////
+//
+//  Tests whether the given token sequence consists only of whitespace
+//  and placemarkers
+//
+///////////////////////////////////////////////////////////////////////////////
+template <typename ContainerT>
+inline bool
+is_blank_only (ContainerT const &argument)
```

Comment:
@hkaiser would love your thoughts here. The purpose of this function is to help detect "no variadic argument supplied" cases. I began with
1. ellipsis index == argument count (so no argument present)

but found I had to add

2. argument consists only of whitespace (present but empty?)

and then modify it to

2. argument consists only of whitespace or placemarkers (thus "is blank")

to get it to work, and that just seems hacky.

Reply:
@jefftrull I understand. IIUC, this function is currently used in a context where you want to decide whether a placemarker has to be inserted or not. However, `is_blank_only` will return `true` even if there already is a placemarker, causing for another one to be inserted into the replacement list. 

I think using `is_whitespace_only` in this context would be sufficient as you want to insert a placemarker whenever there isn't anything but whitespace in the processed sequence.

Reply:
I think this is resolved as discussions above. Thanks!